dataprofiler/profilers/profiler_options.py

```diff
@@ -982,6 +982,64 @@ def _validate_helper(self, variable_path: str = "CorrelationOptions") -> list[st
         return errors
 
 
+class HyperLogLogOptions(BooleanOption):
+    """Options for alternative method of gathering unique row count."""
+
+    def __init__(self, is_enabled: bool = False, seed: int = 0) -> None:
+        """
+        Initialize options for the hyperloglog method of gathering unique row count.
+
+        :ivar is_enabled: boolean option to enable/disable.
+        :vartype is_enabled: bool
+        :ivar seed: seed used to set HLL hashing function
+        :vartype seed: int
+        """
+        BooleanOption.__init__(self, is_enabled=is_enabled)
+        self.seed = seed
```

Comment:
should I also include "p" which represents the 2^p registers in the HLL table

Reply:
Yes per conversation with the team we should implement and option for this